modules/ROOT/pages/dataweave-cookbook-change-script-output-mime.adoc

```diff
@@ -0,0 +1,113 @@
+= Change a Script's MIME Type Output
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, dataweave, data weave, output structure, mime type, mime
+
+This DataWeave example shows how to differentiate the output MIME type from the MIME type in which the output data is formatted. Such differentiation enables a Mule application to output a MIME type that clients can use to determine how to render the application's output data. An example is HTTP, which uses the MIME type to generate its `Content-Type` header.
```

Comment:
Does "this DataWeave example" refer to only the one following or to the whole topic?

Reply:
fixed